test/integration/scheduler/plugins_test.go

```diff
@@ -0,0 +1,57 @@
+package scheduler
```

Comment:
pull-kubernetes-verify job is failed like:
```
Boilerplate header is wrong for: /home/prow/go/src/k8s.io/kubernetes/test/integration/scheduler/plugins_test.go
```
I think that requires the apache license header lines in the file like the other go files.